sources/packages/backend/apps/test-db-seeding/src/db-seeding/02-institution/05-manage-designations.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { DesignationAgreement, InstitutionLocation } from "@sims/sims-db";
+import {
+  createFakeDesignationAgreement,
+  createMultipleFakeInstitutionLocations,
+} from "@sims/test-utils";
+import { Repository } from "typeorm";
+import {
+  DataSeed,
+  DataSeedMethod,
+  SeedPriorityOrder,
+} from "../../seed-executors";
+import { InstitutionHelperService } from "../../services";
+import { SIMS_COLL_F_LEGAL_SIGNING_USER } from "../constants";
+
+@Injectable()
+@DataSeed({ order: SeedPriorityOrder.Unknown })
```

Comment:
What does unknown order means? it is as good as not having an order at all?

Reply:
`unknown` priority is all methods other than the first priority order. 
Actually, there is no need to passes `unknown order`. by default it is `unknown`. I will remove it.